naughty/fedora-35/2323-rhsm-crashes-3

```diff
@@ -0,0 +1,3 @@
+  File "check-system-terminal", line *, in testBasic
+    wait_line(n, prompt)
+*actual text: [systemd]
```

Comment:
This is still not the most robust one as if different systemd service starts to fail, it will reuse the same pattern.

Reply:
Ok, convinced!  Let's modify wait_line to output some context and match on that.